### PR TITLE
Feat: Add auto-labelling to PRs

### PR DESCRIPTION
**Pull request by teolemon**
### What
Add auto-labelling to PRs
### Note
this PR has been autolabelled by this PR. 🤯

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+tests:
+- test/**/*
+
+Template::Toolkit:
+- templates/**/*
+
+Attributes:
+- lib/ProductOpener/Attributes.pm
+
+DataQuality:
```
Comment (teolemon) [inline .github/labeler.yml:19]:
```suggestion
data quality:
```

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+- lib/ProductOpener/Images.pm
+
+Import:
+- lib/ProductOpener/Import.pm
+- lib/ProductOpener/ImportConvert.pm
+
+Ingredients:
+- lib/ProductOpener/Ingredients.pm
+
+KnowledgePanels:
```
Comment (teolemon) [inline .github/labeler.yml:58]:
```suggestion
knowledge-panels:
```

File: .github/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+Numbers:
+- lib/ProductOpener/Numbers.pm
+
+NutriScore:
+- lib/ProductOpener/Nutriscore.pm
+
+Orgs:
+- lib/ProductOpener/Orgs.pm
+
+Packager Codes:
```
Comment (teolemon) [inline .github/labeler.yml:79]:
```suggestion
packager codes:
```

File: .github/labeler.yml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+Export:
+- lib/ProductOpener/Export.pm
+
+Food.pm:
+- lib/ProductOpener/Food.pm
+
+FoodGroups:
+- lib/ProductOpener/FoodGroups.pm
+
+Forest Footprint:
```
Comment (teolemon) [inline .github/labeler.yml:39]:
```suggestion
forest footprint:
```

File: .github/labeler.yml [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+Ecoscore:
+- lib/ProductOpener/Ecoscore.pm
+
+Export:
+- lib/ProductOpener/Export.pm
+
+Food.pm:
+- lib/ProductOpener/Food.pm
+
+FoodGroups:
```
Comment (teolemon) [inline .github/labeler.yml:36]:
```suggestion
Food groups:
```

File: .github/labeler.yml [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+DataQuality:
+- lib/ProductOpener/DataQuality.pm
+- lib/ProductOpener/DataQualityCommon.pm
+- lib/ProductOpener/DataQualityFood.pm
+
+Display:
+- lib/ProductOpener/Display.pm
+
+Ecoscore:
```
Comment (teolemon) [inline .github/labeler.yml:27]:
```suggestion
ecoscore:
```

File: .github/labeler.yml [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+Food.pm:
+- lib/ProductOpener/Food.pm
+
+Food groups:
+- lib/ProductOpener/FoodGroups.pm
+
+forest footprint:
+- lib/ProductOpener/ForestFootprint.pm
+
+GeoIP:
```
Comment (teolemon) [inline .github/labeler.yml:42]:
```suggestion
geolite:
```

File: .github/labeler.yml [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+- lib/ProductOpener/GeoIP.pm
+
+GS1:
+- lib/ProductOpener/GS1.pm
+
+Images:
```
Comment (teolemon) [inline .github/labeler.yml:48]:
```suggestion
images:
```

File: .github/labeler.yml [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+- lib/ProductOpener/Images.pm
+
+Import:
```
Comment (teolemon) [inline .github/labeler.yml:51]:
```suggestion
data import:
```

**Review by alexgarel — COMMENTED** [review]
Great Idea !

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**